extension/content/components/recipes/details/DetailsHeader.js

```diff
@@ -136,6 +142,21 @@ export default function DetailsHeader() {
         >
           Edit Recipe
         </IconButton>
+        <Whisper
+          placement="bottomEnd"
+          speaker={
+            <Popover>
+              <Button appearance="subtle" onClick={handleCopyClick}>
+                Clone Experiment
+              </Button>
+            </Popover>
+          }
+          trigger="click"
+        >
+          <div className="ml-1">
+            <IconButton icon={<Icon icon="ellipsis-h" />} />
+          </div>
```

Comment:
nit: I don't think this `div` adds any value I'd move the `className` to the button.